core/genesis.go

```diff
@@ -291,6 +258,11 @@ func (e *GenesisMismatchError) Error() string {
 type ChainOverrides struct {
 	OverrideCancun *uint64
 	OverrideVerkle *uint64
+	// optimism
+	OverrideOptimismCanyon *uint64
+	// [kroma unsupported]
+	// ApplySuperchainUpgrades bool
+
 	// kroma
 	OverrideKroma *bool
```

Comment:
Why don't we remove `OverrideKroma` and set `OverrideOptimismCanyon` in `makeFullNode`?